waspc/run.ps1

```diff
@@ -22,6 +22,9 @@ switch ($Command) {
     "build:all" {
         Invoke-Expression $BUILD_ALL_CMD
     }
+    "build:packages" {
+        Invoke-Expression $WASP_PACKAGES_COMPILE
+    }
```

Comment:
What feels slightly strange:
 - `build` builds just Haskell
 - `build:packages` builds just packages
 - `build:all` builds Haskell and packages

I suggest changing `build:all` to just `build`, and `build` to `build:hs`. Not really the scope of this PR so you can skip it. RAW.